README.md

```diff
@@ -83,11 +83,12 @@ absl::variant<ConstraintInfo, std::vector<absl::Status>> P4ToConstraintInfo(
 /* p4_constraints/backend/interpreter.h */
 
 // Checks if a given table entry satisfies the entry constraint attached to its
-// associated table. Returns true if this is the case or if no constraint
-// exists. Returns an InvalidArgument Status if the entry belongs to a table not
-// present in ConstraintInfo, or if it is inconsistent with the table definition
-// in ConstraintInfo.
-gutils::StatusOr<bool> EntryMeetsConstraint(const p4::v1::TableEntry& entry,
+// associated table. Returns the empty string if this is the case, or a
+// human-readable nonempty string explaining why it is not the case otherwise.
+// Returns an InvalidArgument Status if the entry belongs to a table
+// not present in ConstraintInfo, or if it is inconsistent with the table
+// definition in ConstraintInfo.
+gutils::StatusOr<std::string> ReasonEntryViolatesConstraint(const p4::v1::TableEntry& entry,
```

Comment:
Nit: Can we fit this into 80 cols?
Perhaps add a line break after the opening `(` and put the arguments on separate lines with 2 or 4 spaces of indent each?

Reply:
Done.